### PR TITLE
docs(folder-structure): improvements

### PR DESCRIPTION
**Pull request by nexxeln**
Closes #<issue>

## ✅ Checklist

- [x] I have followed every step in the [contributing guide](https://github.com/t3-oss/create-t3-app/blob/main/CONTRIBUTING.md) (updated 2022-10-06).
- [x] The PR title follows the convention we established [conventional-commit](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] I performed a functional test on my final commit

---

## Changelog

Made it less verbose and less noobish

---

### CODE REVIEW

File: www/src/pages/en/folder-structure.md [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+├─ public
+│  └─ favicon.ico
+└─ src
+   ├─ env
+   ├─ env
+   │  ├─ client.mjs
+   │  ├─ schema.mjs
+   │  └─ server.mjs
+   ├─ pages
+   │  ├─ _app.tsx
```
Comment (juliusmarminge) [inline www/src/pages/en/folder-structure.md:24]: we also have the index.module.css but who cares

File: www/src/pages/en/folder-structure.md [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+   │     ├─ auth
+   │     │  └─ [...nextauth].ts
+   │     └─ trpc
+   │        └─ [trpc].ts
+   └─ server
+      └─ trpc
+         └─ router
+            ├─ _app.ts
+            ├─ auth.ts
+            └─ example.ts
```
Comment (juliusmarminge) [inline www/src/pages/en/folder-structure.md:55]: move this to the other trpc stuff above

File: www/src/pages/en/folder-structure.md [hunk 3]
```diff
@@ -0,0 +1,240 @@
...
+├─ next-env.d.ts
+├─ next.config.mjs
+├─ package.json
+├─ postcss.config.cjs
+├─ prettier.config.cjs
+├─ tailwind.config.cjs
+├─ tsconfig.json
+```
+
+## `prisma`
```
Comment (juliusmarminge) [inline www/src/pages/en/folder-structure.md:70]: should we have a link like `[See more](/en/usage/prisma)` etc

File: www/src/pages/en/folder-structure.md [hunk 4]
```diff
@@ -0,0 +1,240 @@
...
+
+## `public`
+
+The `public` folder contains static assets that are served by the web server. The `favicon.ico` file is an example of a static asset.
+
+## `src`
+
+The `src` folder contains all the source code of the application.
+
+## `src/env`
```
Comment (juliusmarminge) [inline www/src/pages/en/folder-structure.md:84]:
```suggestion
### `env`
```

maybe make these h3s and skip the parent folder?

File: www/src/pages/en/folder-structure.md [hunk 5]
```diff
@@ -0,0 +1,240 @@
...
+
+### `src/pages/api/auth/[...nextauth].ts`
+
+The `[...nextauth].ts` file is the NextAuth.js authentication slug route. It is used to handle authentication requests. See [NextAuth.js usage](usage/next-auth) for more information on NextAuth.js, and [Next.js Dynamic Routes Docs](https://nextjs.org/docs/routing/dynamic-routes) for info on catch-all/slug routes.
+
+<sub>(with NextAuth.js)</sub>
+
+### `src/pages/api/trpc/[trpc].ts`
+
+The `[trpc].ts` file is the tRPC API entry point. It is used to handle tRPC requests. See [tRPC usage](usage/trpc#-pagesapitrpctrpcts) for more information on this file, and [Next.js Dynamic Routes Docs](https://nextjs.org/docs/routing/dynamic-routes) for info on catch-all/slug routes.
```
Comment (juliusmarminge) [inline www/src/pages/en/folder-structure.md:106]: entrypoint

File: www/src/pages/en/folder-structure.md [hunk 6]
```diff
@@ -0,0 +1,240 @@
...
+
+## `src/utils`
+
+The `utils` folder is used to store commonly re-used utility functions.
+
+<sub>(with tRPC)</sub>
+
+### `src/utils/trpc.ts`
+
+The `trpc.ts` file is the front-end entry point to tRPC. See [tRPC usage](usage/trpc#-utilstrpcts) for more information.
```
Comment (juliusmarminge) [inline www/src/pages/en/folder-structure.md:202]: entrypoint

File: www/src/pages/en/folder-structure.md [hunk 7]
```diff
@@ -0,0 +1,240 @@
...
+
+<sub>(with tRPC)</sub>
+
+## `.env`
+
+The `.env` file is used to store environment variables. See [Environment variables](usage/env-variables) for more information. This file should **not** be commited to git history.
+
+## `.env.example`
+
+The `.env.example` file shows example environment variables based on the chosen libraries. This file should be commited to git history.
```
Comment (c-ehrlich) [inline www/src/pages/en/folder-structure.md:212]: This file is called .env-example currently

**Review by juliusmarminge — COMMENTED** [review]
I don't know about all the config files, is that really necessary?

**Review by c-ehrlich — CHANGES_REQUESTED** [review]

File: www/src/pages/en/folder-structure.md (hunk 7 above)
Comment (juliusmarminge) [reply www/src/pages/en/folder-structure.md:212]: should change that then, .env.example is better imo

File: www/src/pages/en/folder-structure.md [hunk 8]
```diff
@@ -0,0 +1,240 @@
...
+├─ .eslintrc.json
+├─ .gitignore
+├─ README.md
+├─ next-env.d.ts
+├─ next.config.mjs
+├─ package.json
+├─ postcss.config.cjs
+├─ prettier.config.cjs
+├─ tailwind.config.cjs
+├─ tsconfig.json
```
Comment (juliusmarminge) [inline www/src/pages/en/folder-structure.md:67]:
```suggestion
└─ tsconfig.json
```

File: www/src/pages/en/folder-structure.md (hunk 7 above)
Comment (nexxeln) [reply www/src/pages/en/folder-structure.md:212]: > should change that then, .env.example is better imo

Are we doing this before the release

File: www/src/pages/en/folder-structure.md (hunk 4 above)
Comment (nexxeln) [reply www/src/pages/en/folder-structure.md:84]: I liked this way of the whole file being there tbh, but i removed the useless headers

File: www/src/pages/en/folder-structure.md (hunk 7 above)
Comment (c-ehrlich) [reply www/src/pages/en/folder-structure.md:203]: yea i agree calling it .env.example is better

File: www/package.json [hunk 9]
```diff
@@ -46,6 +46,7 @@
     "react": "18.0.0",
     "react-dom": "18.0.0",
     "react-typist": "^2.0.5",
+    "rehype-external-links": "^2.0.1",
```
Comment (c-ehrlich) [inline www/package.json:49]: i think it wasn't removed from the source package.json?

Comment (nexxeln) [reply www/package.json:49]: sorry forgot to commit that

**Review by juliusmarminge — APPROVED** [review]

**Pull request merged**